lib/get-ajv-error-messages.js

```diff
@@ -42,12 +42,33 @@ const getDetailsPerAjvKeyword = {
  * @returns {String} A human-readable validation error message.
  */
 module.exports = function getAjvErrorMessages(ajvErrors, rootName = `root`) {
+  console.log(ajvErrors);
   const errors = ajvErrors.filter(error => !(`propertyName` in error));
 
   return errors.map(error => {
     const getDetails = getDetailsPerAjvKeyword[error.keyword] || (() => error.message);
     const details = getDetails(error, ajvErrors);
-    const errorMessage = `${rootName}${error.dataPath} ${details}`;
-    return errorMessage.replace(`\n`, `\\n`);
+    const errorMessage = `${rootName}${error.dataPath}${getDataDescription(error.data)} ${details}`;
+    return errorMessage.replace(/\n/g, `\\n`);
   });
 };
+
+/**
+ * @param {*} data Any kind of data; not all types can be represented.
+ * @returns {String} A short representation of the given data with a leading space. Empty if this is not possible.
+ */
+function getDataDescription(data) {
+  if (data.constructor === String) {
+    if (data.length > 16) {
+      return ` "${data.substring(0, 15)}…"`;
+    }
+    return ` "${data}"`;
+  }
+  if (data.constructor === Number || data.constructor === Boolean) {
+    return ` ${data}`;
+  }
+  if (data instanceof Object && data.type) {
```

Comment:
Again, also better check for `type` like this:

```suggestion
  if (typeof data === `object` && `type` in data) {
```